packages/typespec-ts/src/modular/helpers/classicalOperationHelpers.ts

```diff
@@ -176,27 +181,41 @@ export function getClassicalOperation(
   if (existFunction) {
     const returnStatement = existFunction.getBodyText();
     if (returnStatement) {
-      let statement = `,
+      let statement: string | undefined = undefined;
+      if (layer !== prefixes.length - 1) {
+        const name = normalizeName(
+          prefixes[layer + 1] ?? "FIXME",
+          NameType.Property
+        );
+
+        // HACK: check if the statement includes a group of this name already to prevent an operation group appearing multiple times
```

Comment:
I feel like it's strange if an operation group would appear multiple times. how could I repro?

Reply:
Same issue as above -- without this the multipart spec generates the group 3 times. I think it might have to do with the way that the operations are nested?